applications/serial_lte_modem/src/slm_at_fota.c

```diff
@@ -375,15 +435,67 @@ void slm_fota_post_process(void)
 
 void slm_finish_modem_fota(int modem_lib_init_ret)
 {
-	if (handle_nrf_modem_lib_init_ret(modem_lib_init_ret)) {
-		char buf[40];
+	char buf[40];
+#if defined(CONFIG_SLM_FULL_FOTA)
```

Comment:
Actually I take this change back to the original version. There are some ram consuming varibles which are used in the code under #ifdef. Since we are running out of ram, it is good to reserve those variables only with the full fota overlay.